pilot/pkg/model/sidecar.go

```diff
@@ -474,6 +474,15 @@ func (sc *SidecarScope) Services() []*Service {
 	return sc.services
 }
 
+func (sc *SidecarScope) DestinationRuleByName(name string) *config.Config {
```

Comment:
I think we should limit SidecarScope's method calling, PushContext should be the right place. Otherwise, new contributors will be confused

Reply:
do you mean `*model.Proxy`

Reply:
I mean move the method to PushContext